src/style.css

```
@@ -1,3 +1,105 @@
+/* @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap'); */
+
 body {
   margin: 0;
+  --header-height: 60px;
+  /* background-color: gainsboro; */
+}
+
+ul {
+  padding: 0;
+  margin: 0;
+}
+
+nav {
+  padding: 0;
+  margin: 0 51px 0 0;
```

```suggestion
  margin: 0 50px 0 0;
```

And better add padding: 0 50px to header and remove this margin